components/KidProfile/index.js

```diff
@@ -0,0 +1,45 @@
+import Image from "next/image";
+import { StyledKidCard, StyledEditButton, StyledBackLink, StyledList } from "./KidProfile.styled";
+import { formatDate } from "../../resources/dateUtils";
+import EventForm from "../EventForm/index";
+import useSWR from "swr";
+import { useRouter } from "next/router";
+
+export default function KidProfile() {
+  const router = useRouter();
+  const { id } = router.query;
+
+  const { data, isLoading, mutate} = useSWR(`/api/kids/${id}`);
+  
+  if (isLoading) {
+    return <div>Loading...</div>;
+  }
+
+  if (!data) {
+    return <div>Where are the kids???</div>;
+  }
+
+ {
+  return (
+      <StyledKidCard>
+        <Image
+          width={200}
+          height={200}
+          src={data.image}
+          alt="Kid Photo"
+        />
+        <h2>{data.name} </h2>
+        <h3>* {formatDate(data.birthDate)}</h3>  
+      <StyledList>
+      {data?.events?.map((event) => (
+          <li key={event._id}>
+           <p>{event.title}</p>
+           <p>{formatDate(event.date)}</p>
+        {/*   <p>{event.image}</p> */}
+          </li>))}
+      </StyledList>
+      <EventForm kidData={data}/>  
```

Comment:
Instead of `kidData` you pass down the _handle function_ for the submit event.

Why? The form doesn't show any data. That's why you only need to let the form know what to do when the submit event is fired.